libs/designsystem/accordion/src/accordion-item.component.ts

```diff
@@ -24,14 +29,28 @@ let uniqueId = 0;
     ]),
   ],
 })
-export class AccordionItemComponent implements OnChanges {
+export class AccordionItemComponent implements OnChanges, AfterContentInit {
   @Input() title: string;
   @Input() isExpanded: boolean = false;
   @Input() isDisabled: boolean = false;
   @Input() disabledTitle: string;
   @Input() hasPadding: boolean = true;
   @Output() toggle: EventEmitter<boolean> = new EventEmitter<boolean>();
 
+  @ContentChildren(ListComponent) kirbyListChildren: QueryList<ListComponent>;
+
+  constructor(private renderer: Renderer2) {}
+
+  ngAfterContentInit(): void {
+    if (this.kirbyListChildren.length > 0) {
+      this.hasPadding = false;
+      this.kirbyListChildren.forEach((child) => {
+        child.shape = 'none';
+        this.renderer.addClass(child.elem.nativeElement, 'transparent');
```

Comment:
Instead of adding a class and implementing the behavior in list-item.component.scss, you could set some class "has-list" with a hostbinding or so, on your accordion item, and do something like this. The CSS property will pierce through encapsulation etc.:

```
:host(.has-list) {
  --kirby-item-background: transparent;
}
```

This way, consumers can always overwrite the --kirby-item-background locally in their accordion-item, if they need some other behavior.

Reply:
I love this idea !! Will try it out 🕺